test/functional/pdo_sqlsrv/pdo_ae_insert_pdoparam_numeric.phpt

```diff
@@ -1,39 +1,123 @@
 --TEST--
-Test for inserting and retrieving encrypted data of numeric types
+Test for inserting encrypted data into numeric types columns
 --DESCRIPTION--
-Use PDOstatement::bindParam with all PDO::PARAM_ types
+Test conversions between different numeric types
+With Always Encrypted, implicit conversion works if:
+1. From input of PDO::PARAM_BOOL to a real column
+2. From input of PDO::PARAM_INT to a any numeric column
+3. From input of PDO::PARAM_STR to a any numeric column
+4. From input of PDO::PARAM_LOB to a any numeric column
+Without Always Encrypted, all of the above works except for input of PDO::PARAM_STR to a bigint column in a x86 platform
+PDO::PARAM_STR does not work for bigint in a x86 because in a x86 platform, the maximum value of an int is about 2147483647
+Whereas in a x64 platform, the maximum value is about 9E18
+In a x86 platform, when in integer is initialized to be > 2147483647, PHP implicitly change it to a float, represented by scientific notation
+When inserting a scientific notation form numeric string, SQL Server returns a converting data type nvarchar to bigint error
```

Comment:
when 'an' integer

Reply:
Anyway, your comment is very thorough, but sounds cumbersome. Is it possible to rephrase some sentences?